### PR TITLE
PR settings

### PR DESCRIPTION
**Pull request by sowmyaraj92**
Assign merge checks for a pull request before approving changes.

### CODE REVIEW

File: src/pages/docs/collaborating-in-postman/version-control-for-collections.md [hunk 1]
```diff
@@ -62,6 +63,34 @@ If your pull request has no conflicts, you can go ahead and open it for review.
 
 Reviewers can [comment on your pull request or decide to merge](#reviewing-pull-requests) your changes into the parent collection.
 
+## Pull request settings
+
+If you are on a business or enterprise plan, the pull request settings will be available in the __Manage roles__ section of a collection.
```
Comment (claireriley) [inline src/pages/docs/collaborating-in-postman/version-control-for-collections.md:68]: I would make this a bit more direct, like the first sentence [here](https://learning.postman.com/docs/designing-and-developing-your-api/monitoring-your-api/setting-up-monitor/#using-static-ip).

Also want to capitalize Business and Enterprise

File: src/pages/docs/collaborating-in-postman/version-control-for-collections.md [hunk 2]
```diff
@@ -62,6 +63,34 @@ If your pull request has no conflicts, you can go ahead and open it for review.
...
+
+<img alt="Collection Manage Roles" src="https://assets.postman.com/postman-docs/collection-manage-roles.jpg" width="250px"/>
+
+In the Postman app, select the collection in the __Collections__ sidebar and click the __View more actions (...)__ button. Select __Manage roles__. Choose __Editor__ option for the users you want to provide editor access to.
```
Comment (claireriley) [inline src/pages/docs/collaborating-in-postman/version-control-for-collections.md:72]: `click the __View more actions (...)__ button.`

->

`click __View more actions (...)__.`

Comment (claireriley) [inline src/pages/docs/collaborating-in-postman/version-control-for-collections.md:72]: I would combine the last two sentences to be 

`Select __Manage roles__, then select __Editor__ for the users you want to provide editor access to`

File: src/pages/docs/collaborating-in-postman/version-control-for-collections.md [hunk 3]
```diff
@@ -62,6 +63,34 @@ If your pull request has no conflicts, you can go ahead and open it for review.
...
+
+[![manage roles](https://assets.postman.com/postman-docs/manage-roles-merge.gif)](https://assets.postman.com/postman-docs/manage-roles-merge.gif)
+
+You need to have __Editor__ access on a collection to merge changes. If you have __Viewer__ access to a collection, you will see a warning icon while adding the reviewers to a pull request.
```
Comment (claireriley) [inline src/pages/docs/collaborating-in-postman/version-control-for-collections.md:76]: I might use "must" instead of "need to" here, optional though

Comment (claireriley) [inline src/pages/docs/collaborating-in-postman/version-control-for-collections.md:76]: Remove "the" before "reviewers"

File: src/pages/docs/collaborating-in-postman/version-control-for-collections.md [hunk 4]
```diff
@@ -62,6 +63,34 @@ If your pull request has no conflicts, you can go ahead and open it for review.
...
+
+<img alt="Reviewer permission" src="https://assets.postman.com/postman-docs/reviewer-permission.jpg" width="850px"/>
+
+Once you've created the pull request, you can assign merge checks before approving changes.
```
Comment (claireriley) [inline src/pages/docs/collaborating-in-postman/version-control-for-collections.md:80]: Remove contraction -> "you've" to "you have"

File: src/pages/docs/collaborating-in-postman/version-control-for-collections.md [hunk 5]
```diff
@@ -62,6 +63,34 @@ If your pull request has no conflicts, you can go ahead and open it for review.
...
+
+<img alt="Merge Check" src="https://assets.postman.com/postman-docs/merge-check-screen.jpg" width="850px"/>
+
+You have three different types of checks that you can enable for a pull request, and they are:
+
+* __Approved once__: You need atleast one approval to merge the pull request.
+* __Approved by a collection editor__: You require the approval of a collection editor to merge the pull request.
+* __Up to date with all changes to this collection__ : You can't merge the pull request if a forked branch is behind the destination branch.
```
Comment (claireriley) [inline src/pages/docs/collaborating-in-postman/version-control-for-collections.md:88]: Remove contraction

File: src/pages/docs/collaborating-in-postman/version-control-for-collections.md [hunk 6]
```diff
@@ -62,6 +63,34 @@ If your pull request has no conflicts, you can go ahead and open it for review.
...
+
+You need to have __Editor__ access on a collection to merge changes. If you have __Viewer__ access to a collection, you will see a warning icon while adding the reviewers to a pull request.
+
+<img alt="Reviewer permission" src="https://assets.postman.com/postman-docs/reviewer-permission.jpg" width="850px"/>
+
+Once you've created the pull request, you can assign merge checks before approving changes.
+
+<img alt="Merge Check" src="https://assets.postman.com/postman-docs/merge-check-screen.jpg" width="850px"/>
+
+You have three different types of checks that you can enable for a pull request, and they are:
```
Comment (claireriley) [inline src/pages/docs/collaborating-in-postman/version-control-for-collections.md:84]: I would adjust this to make it a bit more succinct

> There are three different types of checks that you can enable for a pull request:

File: src/pages/docs/collaborating-in-postman/version-control-for-collections.md [hunk 7]
```diff
@@ -62,6 +63,34 @@ If your pull request has no conflicts, you can go ahead and open it for review.
...
+
+* __Approved once__: You need atleast one approval to merge the pull request.
```
Comment (claireriley) [inline src/pages/docs/collaborating-in-postman/version-control-for-collections.md:86]: Add space

Comment (claireriley) [inline src/pages/docs/collaborating-in-postman/version-control-for-collections.md:86]: Could we change "you" to "users" in this list? Would that make sense? or "team members" or "viewers" if applicable

File: src/pages/docs/collaborating-in-postman/version-control-for-collections.md [hunk 8]
```diff
@@ -62,6 +63,34 @@ If your pull request has no conflicts, you can go ahead and open it for review.
...
+* __Approved by a collection editor__: You require the approval of a collection editor to merge the pull request.
+* __Up to date with all changes to this collection__ : You can't merge the pull request if a forked branch is behind the destination branch.
+
+> If you do not have editor access to the collection, the __Merge__ icon will be disabled.
```
Comment (claireriley) [inline src/pages/docs/collaborating-in-postman/version-control-for-collections.md:90]: `the __Merge__ icon will be disabled` -> `__Merge__ will be disabled`

**Review by claireriley — CHANGES_REQUESTED** [review]
Looks good! Few suggested changes

**Review by sowmyaraj92 — COMMENTED** [review]
Hey @claireriley, I have made the requested changes : )

File: src/pages/docs/collaborating-in-postman/version-control-for-collections.md [hunk 9]
```diff
@@ -62,6 +63,34 @@ If your pull request has no conflicts, you can go ahead and open it for review.
 
 Reviewers can [comment on your pull request or decide to merge](#reviewing-pull-requests) your changes into the parent collection.
 
+## Pull request settings
+
+Pull request settings are available on Postman Business and Enterprise plans in the __Manage roles__ section of a collection.
```
Comment (claireriley) [inline src/pages/docs/collaborating-in-postman/version-control-for-collections.md:68]: Small thing but I'd capitalize "roles" like how it is in the screenshot

File: src/pages/docs/collaborating-in-postman/version-control-for-collections.md [hunk 10]
```diff
@@ -65,29 +65,29 @@ Reviewers can [comment on your pull request or decide to merge](#reviewing-pull-
...
 
 <img alt="Merge Check" src="https://assets.postman.com/postman-docs/merge-check-screen.jpg" width="850px"/>
 
-You have three different types of checks that you can enable for a pull request, and they are:
+There are three different types of checks that you can enable for a pull request:
 
-* __Approved once__: You need atleast one approval to merge the pull request.
-* __Approved by a collection editor__: You require the approval of a collection editor to merge the pull request.
-* __Up to date with all changes to this collection__ : You can't merge the pull request if a forked branch is behind the destination branch.
+* __Approved once__ : You need atleast one approval to merge the pull request.
```
Comment (claireriley) [inline src/pages/docs/collaborating-in-postman/version-control-for-collections.md:86]: add space to "atleast"

File: src/pages/docs/collaborating-in-postman/version-control-for-collections.md [hunk 11]
```diff
@@ -65,29 +65,29 @@ Reviewers can [comment on your pull request or decide to merge](#reviewing-pull-
...
+* __Approved by a collection editor__ : You require the approval of a collection editor to merge the pull request.
+* __Up to date with all changes to this collection__ : You cannot merge the pull request if a forked branch is behind the destination branch.
 
-> If you do not have editor access to the collection, the __Merge__ icon will be disabled.
+> If you do not have editor access to the collection, the __Merge__ will be disabled.
```
Comment (claireriley) [inline src/pages/docs/collaborating-in-postman/version-control-for-collections.md:90]: I'm wondering if "the __Merge__" might be better stated as "the option to __Merge__" ?

**Review by claireriley — APPROVED** [review]
Hey @sowmyaraj92 ! Few very small edits, I'll approve now so you'll be good to merge once you've made those.

**Comment by claireriley** [comment]
@sowmyaraj92 are we ready to merge this one or are we waiting on a release?

**Comment by sowmyaraj92** [comment]
> @sowmyaraj92 are we ready to merge this one or are we waiting on a release?

@claireriley We are waiting for the release, minor enhancement is coming up for this- will add that as well before we go ahead and merge.

**Comment by sowmyaraj92** [comment]
> > @sowmyaraj92 are we ready to merge this one or are we waiting on a release?
> 
> @claireriley We are waiting for the release, minor enhancement is coming up for this- will add that as well before we go ahead and merge.

@claireriley I have updated the flow and screenshots, do give it a quick look - need to merge the PR as this is up on production.

**Pull request merged**